libs/core/src/lib/input-group/input-group.component.ts

```diff
@@ -262,6 +266,11 @@ export class InputGroupComponent implements ControlValueAccessor, OnInit, OnDest
         this.addOnButtonClicked.emit($event);
     }
 
+    /** @hidden */
+    onSearchEvent($event: Event) {
```

Comment:
linter will complain for the absence of return type
```suggestion
    onSearchEvent(event: Event): void {
```